docs/dev-guide/contributing/building.rst

```diff
@@ -77,12 +82,16 @@ Koji tags. For example, if ``python-celery-3.1.11-1.el7.x86_64`` is built into t
 must indicate the build to use in the version field of the release stream's definition file,
 ``2.5-dev.yaml`` in this case.
 
-Because there is no way to automatically determine when a particular component needs to be rebuilt or what that version should be, the build-infrastructure assumes that whatever version is specified
+Because there is no way to automatically determine when a particular component needs to be rebuilt
+or what that version should be, the build-infrastructure assumes that whatever version is specified
 in the yaml file is the final version that is required.  If a release build of that version had
 already been built in koji then those RPMs will be used. If the version specified in the yaml file
 does not match the version in the spec file, the spec file will be updated and the change will be
 merged forward using the 'ours' strategy.
 
+Information about these release configs, and which config to use for building, can be found in the
+config README: https://github.com/pulp/pulp_packaging/blob/master/ci/config/releases/README.md
```

Comment:
This one particularly would be good as a link I think, where others maybe not so much.